s3po/backends/s3.py

```diff
@@ -26,7 +26,7 @@ def download(self, bucket, key, destination, retries, headers=None):
         bucket = self.get_bucket(bucket)
         # Make a file that we'll write into
         destination = CountFile(destination)
-        obj = bucket.get_key(key)
+        obj = bucket.get_key(key, validate=False)
```

Comment:
So, I think this will never cause the following exception to be thrown. I don't know if anyone is relying on this exception or not, but it seems like that might be something worth checking.